components/nugetAlias.txt

```diff
@@ -88,4 +88,10 @@ Extensions.Logging: NServiceBus.Extensions.Logging
 TimeoutMigration: Particular.TimeoutMigration
 CosmosDB: NServiceBus.Persistence.CosmosDB
 NonDurablePersistence: NServiceBus.Persistence.NonDurable
-TransactionalSession: NServiceBus.TransactionalSession
\ No newline at end of file
+TransactionalSession: NServiceBus.TransactionalSession
```

Comment:
This is just a thought. Do we consider grouping the tx session related stuff more important or the fact that persistence and tx session go hand in hand? For the later, we could add the persistence-specific component alias to the persistence. Same for the component definitions

Reply:
I have no strong opinion. Happy to move it around if rest of you folks agree.

Reply:
Me neither. I'm good either way. I just wanted to share my observations and thoughts